airbyte-integrations/connectors/source-elasticsearch/src/main/java/io/airbyte/integrations/source/elasticsearch/ElasticsearchConnection.java

```diff
@@ -87,9 +87,25 @@ protected Header[] configureHeaders(ConnectorConfiguration config) {
         headerList.add(new BasicHeader("Authorization", basicHeader));
       }
     }
+
+    headerList.add(new BasicHeader("user-agent", this.getConnectorVersion()));
+
     return headerList.toArray(new Header[headerList.size()]);
   }
 
+  /**
+   * Read the version of the connector from the environment
+   * @return the version of the connector
+   */
+  private String getConnectorVersion() {
```

Comment:
Same as the destination one — naming perhaps can be more explicit, `makeConnectorUAString`? Not attached to a specific name.